.gitattributes

```diff
@@ -200,4 +200,12 @@ makefile       text
 ## EXECUTABLES
 *.pyc binary
 
-
+## LINGUST OVERRIDES
+/Dockerfile.dev linguist-language=dockerfile
+/docker-compose linguist-language=dockerfile
+/Dockerfile linguist-language=dockerfile
+/components/core/Dockerfile.prod linguist-language=dockerfile
+/ui/.bowerrc linguist-language=json
+/ui/bower.json linguist-language=json
+/components/frontend-react/.prettierrc linguist-language=json
```

Comment:
You no need to styles these hidden file for now

Reply:
You missed this.

Reply:
Are you referring to the files that start with '.' as the hidden files?